x-pack/plugins/elastic_assistant/server/routes/capabilities/get_capabilities_route.ts

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { IKibanaResponse, IRouter } from '@kbn/core/server';
+import { transformError } from '@kbn/securitysolution-es-utils';
+
+import { CAPABILITIES } from '../../../common/constants';
+import { ElasticAssistantRequestHandlerContext } from '../../types';
+
+import { GetCapabilitiesResponse } from '../../schemas/capabilities/get_capabilities_route.gen';
```

Comment:
Consider moving `GetCapabilitiesResponse` and the other types generated from the schema (required by both the client and server) to the `kbn-elastic-assistant-common`. 

Per our offline discussion, the current implementation of the openapi code gen scripts must be updated to implement this, because it defaults to outputting the `.gen` types file in the same directory as the source schema.

Reply:
I have this working locally, and after looking at the changset I think it would be best to make this update in a separate PR as an enhancement to the `@kbn-openapi-generator` package. It's a fairly small change and I'll include your feedback here as an initial use/reference example. I'll also fix the `getGeneratedFilePath()` bug you ran into when there's a `.` in the path in that PR as well (to keep all the openapi-generator fixes together).